components/nav/Navbar.vue

```diff
@@ -36,152 +36,251 @@
                 >
+                  <nuxt-img
+                    src="/nav-dropdown/html-form-backend.png"
+                    class="dropdown-item__img"
+                  />
+                  <div class="d-flex flex-column ms-3">
+                    <span class="dropdown-item__title">HTML Form Backend</span>
+                    <span class="dropdown-item__desc"
+                      >Simplify form integration</span
+                    >
+                  </div>
+                </NuxtLink>
               </li>
               <li>
                 <NuxtLink
-                  class="dropdown-item"
+                  class="dropdown-item d-flex align-items-center"
                   to="/features/auto-responder/"
                   @click="collapseNav"
-                  >Auto Responder
+                >
+                  <nuxt-img
+                    src="/nav-dropdown/auto-responders.png"
+                    class="dropdown-item__img"
+                  />
+                  <div class="d-flex flex-column ms-3">
+                    <span class="dropdown-item__title">Auto Responder</span>
+                    <span class="dropdown-item__desc"
+                      >Set up automated responses</span
+                    >
+                  </div>
                 </NuxtLink>
               </li>
               <li>
                 <NuxtLink
-                  class="dropdown-item"
+                  class="dropdown-item d-flex align-items-center"
                   to="/features/spam-protection/"
                   @click="collapseNav"
-                  >Spam Protection</NuxtLink
                 >
+                  <nuxt-img
+                    src="/nav-dropdown/spam-protection.png"
+                    class="dropdown-item__img"
+                  />
+                  <div class="d-flex flex-column ms-3">
+                    <span class="dropdown-item__title">Spam Protection</span>
+                    <span class="dropdown-item__desc"
+                      >Protect sensitive data</span
+                    >
+                  </div>
+                </NuxtLink>
               </li>
               <li>
                 <NuxtLink
-                  class="dropdown-item"
+                  class="dropdown-item d-flex align-items-center"
                   to="/features/powerful-analytics/"
                   @click="collapseNav"
-                  >Powerful Analytics</NuxtLink
                 >
+                  <nuxt-img
+                    src="/nav-dropdown/powerful-analytics.png"
+                    class="dropdown-item__img"
+                  />
+                  <div class="d-flex flex-column ms-3">
+                    <span class="dropdown-item__title">Powerful Analytics</span>
+                    <span class="dropdown-item__desc"
+                      >Get Insights of Engagements</span
+                    >
+                  </div>
+                </NuxtLink>
```

Comment:
This can now be a component in itself and we can loop over a list of navigation items to show the elements.

Reply:
Yes component all the way to make life easier.